test/compiler.test.js

```diff
@@ -662,6 +688,75 @@ describe('compiler', function() {
       });
     });
 
+    it('resolves relative path in `modelSources` option', function() {
+      appdir.createConfigFilesSync({}, {}, {
+        Car: { dataSource: 'db' }
+      });
+      appdir.writeConfigFileSync('custom-models/car.json', { name: 'Car' });
+      var appJS = appdir.writeFileSync('custom-models/car.js', '');
+
+      var instructions = boot.compile({
+        appRootDir: appdir.PATH,
+        modelSources: ['./custom-models']
+      });
+
+      expect(instructions.models).to.have.length(1);
+      expect(instructions.models[0].sourceFile).to.equal(appJS);
+    });
+
+    it('resolves non-relative path in `modelSources` option', function() {
```

Comment:
Please rename to "resolves module relative path..." to keep this consistent with other test names.